trunner/harness/unity.py

```diff
@@ -46,13 +48,17 @@ def unity_harness(dut: Dut, ctx: TestContext) -> Optional[TestResult]:
         parsed = dut.match.groupdict()
 
         if idx == 0:
-            if parsed["status"] == "FAIL":
-                last_fail = parsed
+            if parsed["status"] in ["FAIL", "IGNORE"]:
+                last_assertion = parsed
         elif idx in (1, 2):
-            # TODO we do not consider INFO/IGNORE messages
+            # TODO we do not consider INFO messages
```

Comment:
Maybe do check here `if last_assertion and last_assertion["status"] != "INFO"`. Then we know that last assertion exists and it's not INFO.